lisa/tools/git.py

```diff
@@ -201,6 +201,17 @@ def apply(
         )
         result.assert_exit_code(message=f"failed on applying patches. {result.stdout}")
 
+    def am(self, cwd: pathlib.PurePath, patch: pathlib.PurePath) -> None:
```

Comment:
Change the name to `apply_mail`. But if it's possible, you can check the file ext, and implement it in `apply` method.